qa/tasks/cephfs/test_admin.py

```diff
@@ -2625,3 +2625,103 @@ def test_with_health_warn_with_2_active_MDSs(self):
                               errmsgs=health_warn)
         self.run_ceph_cmd(f'mds fail {mds1_id} --yes-i-really-mean-it')
         self.run_ceph_cmd(f'mds fail {mds2_id} --yes-i-really-mean-it')
+
+class TestToggleVolumes(CephFSTestCase):
+    '''
+    Contains code for enabling/disabling mgr/volumes plugin.
+    '''
+
+    MGR_MOD_NAME = 'volumes'
+    CONFIRM = '--yes-i-really-mean-it'
+
+    def test_enabled_by_default(self):
+        '''
+        Test that volumes plugin is enabled by default and is also reported as
+        "always on".
+        '''
+        output = self.get_ceph_cmd_stdout('mgr module ls')
```

Comment:
Commenting again - this should use JSON format.

Reply:
Okay, re-wrote code to use JSON instead. JSON is available for `module ls` but not for `module enable and `module disable`. So I've left latter 2 unchanged.

Reply:
Done.